regcomp_trie.c

```diff
@@ -816,7 +816,7 @@ Perl_make_trie(pTHX_ RExC_state_t *pRExC_state, regnode *startbranch,
                 svpp = hv_fetch( widecharmap, (char*)&uvc, sizeof( UV ), 1 );
 
                 if ( !svpp )
-                    Perl_croak( aTHX_ "error creating/fetching widecharmap entry for 0x%" UVXf, uvc );
+                    croak("error creating/fetching widecharmap entry for 0x%" UVXf, uvc );
```

Comment:
This should be a panic message for instance.

Reply:
That sounds like a good suggestion for another branch. In this branch I'm aiming not to have any behavioral changes.